api/nakadi-event-bus-api.yaml

```diff
@@ -1217,6 +1221,7 @@ definitions:
       - name
       - category
       - owning_application
+      - enrichment_strategies
```

Comment:
is it possible to create undefined event-type without enrichment_strategies field